src/scenes/AcceleratorRouter/AcceleratorCohortNewView.tsx

```diff
@@ -0,0 +1,83 @@
+import { Typography, useTheme } from '@mui/material';
+import { BusySpinner } from '@terraware/web-components';
+import React, { useCallback, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { APP_PATHS } from 'src/constants';
+import AcceleratorMain from 'src/scenes/AcceleratorRouter/AcceleratorMain';
+import CohortForm from 'src/scenes/AcceleratorRouter/CohortForm';
+import strings from 'src/strings';
+import useForm from 'src/utils/useForm';
+import useSnackbar from 'src/utils/useSnackbar';
+
+export type CreateCohortRequest = {
+  currentPhase: 0 | 1 | 2 | 3;
+  name: string;
+  organizationId: number;
+};
+
+export type UpdateCohortRequest = {
+  currentPhase: 0 | 1 | 2 | 3;
+  name: string;
+  organizationId: number;
+};
```

Comment:
I have a PR out to generate the types, once we have that we can move these into the types directory.

Reply:
@joshbuchea API should now show up in the generated types. Try again.

Reply:
Accelerator API is now showing up, thanks @karthikbtf 🙌